### PR TITLE
internal_windows_advapi: alias syntax update

### PR DESCRIPTION
**Pull request by burner**

### CODE REVIEW

File: std/internal/cstring.d [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿/**
+/**
```
Comment (yebblies) [inline std/internal/cstring.d]: ?

Comment (burner) [inline std/internal/cstring.d]: I did see a space there

Comment (yebblies) [inline std/internal/cstring.d]: I don't see a space there.  Did you accidentally change the line ending?

Comment (burner) [inline std/internal/cstring.d]: I'll have a look

Comment (burner) [inline std/internal/cstring.d]: my bad

**Comment by yebblies** [comment]
Auto-merge toggled on

**Comment by burner** [comment]
thanks

**Pull request merged**